example/AppContext.tsx

```diff
@@ -146,12 +148,13 @@ class AppContextProvider extends React.Component<{}, AppContextState> {
             }));
 
             // Initialize the session
-            const { accessToken } = await auth.authorize(config);
+            const { accessToken, code } = await auth.authorize(config);
+            const token = code || accessToken;
```

Comment:
@reteps Both code and accessToken seem to work to connect to the remote, but perhaps Im misunderstanding the purpose of each.